src/ducks/Signals/chart/VisualBacktestChart.js

```diff
@@ -99,24 +104,32 @@ const VisualBacktestChart = ({
   )
 }
 
-const getDataKeys = (signal = {}) => {
+export const getDataKeys = (signal = {}) => {
   if (signal.active_addresses) {
-    return { metric: 'dailyActiveAddresses', signal: 'active_addresses' }
+    return {
+      metric: 'dailyActiveAddresses',
+      key: 'daily_active_addresses',
+      signal: 'active_addresses'
+    }
```

Comment:
Suggest to use one place of truth - Metrics

Reply:
Good point